
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 115, in <module>
    from sklearn.externals import joblib
ImportError: cannot import name 'joblib'
[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'sklearn.externals.joblib'), ('new_fqn', 'sklearn.experimental'), ('line_no', 115)])
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 122, in <module>
    sklearn.experimental.dump(model,'predicted_fish.pkl')
AttributeError: module 'sklearn' has no attribute 'experimental'
[REPAIR EXEC TIME]: 6.010587930679321
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 115, in <module>
    from sklearn.externals import joblib
ImportError: cannot import name 'joblib'
[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'sklearn.externals.joblib'), ('new_fqn', 'sklearn.experimental'), ('line_no', 115)])
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 122, in <module>
    sklearn.experimental.dump(model,'predicted_fish.pkl')
AttributeError: module 'sklearn' has no attribute 'experimental'
[REPAIR EXEC TIME]: 3.162457227706909
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 115, in <module>
    from sklearn.externals import joblib
ImportError: cannot import name 'joblib'
[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'sklearn.externals.joblib'), ('new_fqn', 'sklearn.experimental'), ('line_no', 115)])
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 122, in <module>
    sklearn.experimental.dump(model,'predicted_fish.pkl')
AttributeError: module 'sklearn' has no attribute 'experimental'
[REPAIR EXEC TIME]: 3.0956802368164062
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 115, in <module>
    from sklearn.externals import joblib
ImportError: cannot import name 'joblib'
[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'sklearn.externals.joblib'), ('new_fqn', 'sklearn.experimental'), ('line_no', 115)])
89.08778355829942 %
Traceback (most recent call last):
  File "kernel49cd9d947d.py", line 122, in <module>
    sklearn.experimental.dump(model,'predicted_fish.pkl')
AttributeError: module 'sklearn' has no attribute 'experimental'
[REPAIR EXEC TIME]: 6.839068651199341